scripts/base_network.py

```diff
@@ -160,6 +157,38 @@ def _set_links_underwater_fraction(fp_offshore_shapes, n):
             )
 
 
+def _set_links_underwater_fraction(lines_or_links, fp_offshore_shapes):
```

Comment:
maybe we could drop the "_links" as the function is now general, what do you think?

Reply:
Absolutely agree. Could we probably replace "_links" with "_dc" to make the function purpose clear?